### PR TITLE
Content API - HTML converters

### PR DESCRIPTION
**Pull request by mharsat**
Both `toHtml` and `fromHtml` support these rich text features:

- headings
- bold
- italic
- links
- lists
- line break
- emojis

Other rich content features like plugins are removed

### CODE REVIEW

File: packages/ricos-content/web/src/converters/html/toHtml/toHtml.ts [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  [Decoration_Type.LINK]: 'a',
+};
+const NODE_TO_HTML_TAG = {
+  [Node_Type.BULLET_LIST]: 'ul',
+  [Node_Type.ORDERED_LIST]: 'ol',
+  [Node_Type.LIST_ITEM]: 'li',
+  [Node_Type.PARAGRAPH]: 'p',
+};
+
+const EMPTY_PARAGRAPH = '<p><br /></p>';
```
Comment (oded-soffrin) [inline packages/ricos-content/web/src/converters/html/toHtml/toHtml.ts:67]: Why <br/> and not a space?

File: packages/ricos-content/web/src/converters/nodeUtils.ts [hunk 2]
```diff
@@ -6,30 +6,51 @@ import {
...
-  key: genKey(),
-  nodes: [],
+  ...createNode(Node_Type.TEXT),
   textData: {
     text,
     decorations,
   },
 });
 
+export const createHeadingNode = (
```
Comment (oded-soffrin) [inline packages/ricos-content/web/src/converters/nodeUtils.ts:38]: @lxgreen does this fits with your branch of content API?

File: packages/ricos-content/web/src/converters/html/toHtml/toHtml.ts (hunk 1 above)
Comment (mharsat) [reply packages/ricos-content/web/src/converters/html/toHtml/toHtml.ts:67]: So we can represent a paragraph without text which is literally a line break. A space character doesn't start a new line. The RichText editor creates new lines similarly (have a look at the html file Alex extracted from their editor in `fromHtml`

File: packages/ricos-content/web/src/converters/nodeUtils.ts (hunk 2 above)
Comment (lxgreen) [reply packages/ricos-content/web/src/converters/nodeUtils.ts:38]: Yep, all these createXXXNode utils are core builder methods of content api

**Review by lxgreen — APPROVED** [review]
👍

**Pull request merged**